docs/Building.md

```diff
@@ -8,6 +8,124 @@ This library fully supports Qt 6.5 and newer.
 Qt 5.15 is fully supported only on desktop platforms,
 previous Qt 5 versions down to 5.6 only support widgets but not Qt Location.
 
+### Distros packaging the necessary Qt6 private headers
```

Comment:
I would move this below to the Linux section. Also maybe call them "Linux distributions" or "Distributions".

Reply:
Done